src/core/public/i18n/i18n_eui_mapping.test.ts

```diff
@@ -90,7 +90,38 @@ describe('@elastic/eui i18n tokens', () => {
           .replace(/\s{2,}/g, ' ')
           .trim();
 
-        expect(i18nTranslateCall[1].defaultMessage).toBe(normalizedDefString);
+        if (!isDefFunction) {
+          expect(i18nTranslateCall[1].defaultMessage).toBe(normalizedDefString);
+        } else {
+          // Certain EUI defStrings are actually functions (that currently primarily handle
+          // pluralization). To check EUI's pluralization against Kibana's pluralization, we
+          // need to eval the defString and then actually i18n.translate & compare the 2 outputs
+          const defFunction = eval(defString); // eslint-disable-line no-eval
+          const defFunctionArg = normalizedDefString.split('({ ')[1].split('})')[0]; // TODO: All EUI pluralization fns currently only pass 1 arg. If this changes in the future and 2 args are passed, we'll need to do some extra splitting by ','
```

Comment:
Just want to highlight this TODO, I'm not sure when we would ever pass multiple args into an EUI pluralization fn - but if we ever do in the future this logic will need to be reworked because all I'm essentially doing here is isolating the text in `'({ someVar })` to get a correct single variable name